tutorials/Bayesian_Logistic_Regression.ipynb

```diff
@@ -3,13 +3,30 @@
   {
```

Comment:
<p>This and subsequent plots do not display in colab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>